core/App/utils/helpers.ts

```diff
@@ -143,6 +143,14 @@ export function getCredentialConnectionLabel(credential?: CredentialExchangeReco
     : credential?.connectionId ?? ''
 }
 
+export function getConnectionImageUrl(connectionId: string) {
```

Comment:
I think this fn can return an actual url, and empty string, or undefined. Might be better to just return a non-empty string or undefined to better K.I.S.S.